### PR TITLE
e2e: add basic bean ref tests

### PR DESCRIPTION
**Pull request by tplevko**

### CODE REVIEW

File: packages/ui/src/components/Form/bean/NewBeanModal.tsx [hunk 1]
```diff
@@ -56,10 +56,10 @@ export const NewBeanModal: FunctionComponent<NewBeanModalProps> = (props: NewBea
         isOpen={props.isOpen}
         onClose={handleCancel}
         actions={[
-          <Button key="confirm" variant="primary" onClick={handleConfirm}>
+          <Button key="confirm" variant="primary" onClick={handleConfirm} data-testid="creat-bean-btn">
```
Comment (igarashitm) [inline packages/ui/src/components/Form/bean/NewBeanModal.tsx:59]: just one typo, `creat` -> `create`

File: packages/ui-tests/cypress/fixtures/flows/SqlBeansCR.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    from:
+      uri: timer
+      steps:
+        - to:
+            uri: sql
+        - to:
+            uri: log:test
+- beans:
+    - name: mysqlDataSource
+      type: io.kaoto.mysqlDataSource
```
Comment (igarashitm) [inline packages/ui-tests/cypress/fixtures/flows/SqlBeansCR.yaml:12]: Trivial but usually the class name starts with a capital letter, so `MysqlDataSource` or `MySQLDataSource` might look nice

**Review by igarashitm — APPROVED** [review]

File: packages/ui/src/components/Form/bean/NewBeanModal.tsx (hunk 1 above)
Comment (tplevko) [reply packages/ui/src/components/Form/bean/NewBeanModal.tsx]: thanks! fixed

File: packages/ui-tests/cypress/fixtures/flows/SqlBeansCR.yaml (hunk 2 above)
Comment (tplevko) [reply packages/ui-tests/cypress/fixtures/flows/SqlBeansCR.yaml]: thanks for noticing! fixed

File: packages/ui-tests/cypress/fixtures/flows/SqlBeansCR.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    - name: mysqlDataSource
+      type: io.kaoto.MysqlDataSource
+      properties:
+        port: "3306"
+        url: jdbc:mysql://localhost/test
+        username: dbuser
+        password: dbpass
+        driver-class-name: com.mysql.jdbc.Driver
+    - name: mysqlPostgreSqlSource
+      type: io.kaoto.mysqlPostgreSqlSource
```
Comment (igarashitm) [inline packages/ui-tests/cypress/fixtures/flows/SqlBeansCR.yaml:20]: Probably this one also to start the class name with a capital letter?

**Pull request merged**